internal/core/command/v2/controller/http/command.go

```diff
@@ -96,6 +98,18 @@ func (cc *CommandController) CommandsByDeviceName(w http.ResponseWriter, r *http
 	pkg.Encode(response, w, lc)
 }
 
+func validateGetCommandParameters(r *http.Request) (err errors.EdgeX) {
+	dsReturnEvent := utils.ParseQueryStringToString(r, v2.ReturnEvent, v2.ValueYes)
+	dsPushEvent := utils.ParseQueryStringToString(r, v2.PushEvent, v2.ValueNo)
+	if dsReturnEvent != v2.ValueYes && dsReturnEvent != v2.ValueNo {
+		return errors.NewCommonEdgeX(errors.KindContractInvalid, "invalid query parameter", nil)
```

Comment:
add more error message
`"invalid query parameter, %s has to be %s or %s", dsReturnEvent, ValueYes, ValueNo`

Reply:
Added error message.